contracts/compiled/OwnableApprovableERC721.json

```diff
@@ -0,0 +1,341 @@
+{
+  "_format": "hh-sol-artifact-1",
+  "contractName": "OwnableApprovableERC721",
```

Comment:
I know this is a generated file but we should get the name to read liquid infrastructure erc721

Reply:
Everything has been renamed to LiquidInfrastructure